tests/unit/notes/test-class-wc-payments-notes-additional-payment-methods.php

```diff
@@ -24,78 +24,6 @@ public function tear_down() {
 		delete_option( '_wcpay_feature_upe' );
 	}
 
-	public function test_get_note() {
```

Comment:
All the non-UPE related tests have been removed from this class, since the plugin now executes UPE by default